src/app/tests/suites/certification/Test_TC_APPLAUNCHER_3_7.yaml

```diff
@@ -57,16 +60,34 @@ tests:
           values:
               - name: "Status"
                 value: 0
+              - name: "Data"
+                value: "DataValue"
```

Comment:
should this be DataValue and not "DataValue" so it references the value of DataValue declared on line 31?

Reply:
wasn't the purpose of defining DataValue on line 31 so that it would be referenced here?